oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s_service/k8s-service-service.json

```diff
@@ -2074,7 +2074,52 @@
       "layer": "K8S_SERVICE",
       "entity": "Service",
       "name": "K8S-Service-Service",
-      "id": "K8S-Service-Service"
+      "id": "K8S-Service-Service",
+      "isDefault": true,
```

Comment:
@songzhendong  This is the key config to fix dashboard popup about https://github.com/apache/skywalking/pull/12072. That is the same.

Reply:
Thank you